roles/openshift_provisioners/tasks/install_provisioners.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Check that EFS File System ID is set
+  fail: msg='the openshift_provisioners_efs_fsid variable is required'
+  when: (openshift_provisioners_efs | bool) and openshift_provisioners_efs_fsid is not defined
```

Comment:
This is the type of verification that would be nice to run at the very beginning of the installation, in order to fail fast (and not half-way through the install).

I don't see a way how to run a "pre-condition" check for multiple roles automatically unless we settle on a name convention and write some code to discover pre-condition-checking files.

Another possibility is to move this precondition checking to a single role that runs early on, e.g. `openshift_sanitize_inventory`. The problem with that approach is the perhaps poor factorization of the code.

@sdodson what are the current thoughts on how to handle this, running pre-condition checks early on?